component/pom.xml

```diff
@@ -40,9 +45,23 @@
 			<version>1.16.8</version>
 			<scope>provided</scope>
 		</dependency>
-
-
-    </dependencies>
+		<dependency>
+			<groupId>javax.jcr</groupId>
+			<artifactId>jcr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.exoplatform.jcr</groupId>
+			<artifactId>exo.jcr.component.core</artifactId>
+			<version>6.2.x-SNAPSHOT</version>
```

Comment:
Could you use variables for version tags?

Reply:
ok